src/Neyro.AppMetrics.Extensions.EventCountersCollector/Neyro.AppMetrics.Extensions.EventCountersCollector/_InternalVisibleTo.cs

```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("UnitTests")]
```

Comment:
What about put it to .csproj?
```
<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
    <_Parameter1>$(AssemblyName).Tests</_Parameter1>
</AssemblyAttribute>
```
or
```
<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
    <_Parameter1>UnitTests</_Parameter1>
</AssemblyAttribute>
```

Reply:
Thanks @neyromant didn't know about you can set internals in csproj. I've updated accordingly